assets/jsx/workflow-editor/utils.jsx

```diff
@@ -317,6 +317,7 @@ export function getStepScopedVariables(node) {
             description: variable?.description,
             nodeSlug: node.data.slug,
             priority: variable?.priority || DEFAULT_VARIABLE_PRIORITY,
+            table: variable?.table
```

Comment:
This make sense for the meta items, but not for all the others. If we look ahead in the future, when we could also have other items with custom attributes, we would end with some empty attributes that make sense only for specific items, or data types. I probably did the same mistake for the Array data type, witht he itemsType property - but we should not fix it on this issue.

What about to remove the table attribute and add a more generic one that works as parent for any other custom attribute?  Something like "context", or maybe "config"?

```javascript
...
context: variable?.context
...
```

and in the data type definition:

```javascript
...
context: {
    table: '_usermeta'
}
...
```

Reply:
This project requires this type of thinking, looking ahead to implement a feature, but do not be tied to it, keeping the structure dynamic for other features or future changes.

Reply:
Thanks @andergmartins Your suggestion make sense